packages/vuetify/src/components/VIcon/VIcon.tsx

```diff
@@ -74,6 +74,7 @@ export const VIcon = genericComponent()({
           ]}
           role={ attrs.onClick ? 'button' : undefined }
           aria-hidden={ !attrs.onClick }
+          tabindex={ attrs.onClick ? '0' : undefined }
```

Comment:
This doesn't work if the user supplies a meta argument like "@click.once", or "@click.stop".

```vue
<template>
  <v-app>
    <div class="ma-4 pa-4">
      <v-icon icon="$vuetify" @click.once="onClick" />
    </div>
  </v-app>
</template>

<script setup>
  function onClick () {
    alert('click')
  }
</script>
```